app/views/pages/home/_confidence.html.erb

```diff
@@ -1,51 +1,51 @@
 <div class="govuk-width-container">
-  <div class="ncce-confidence__header govuk-heading-m">
+  <div class="ncce-confidence__header govuk-heading-s">
     <h1><%= t("pages.home.confidence.heading") %></h1>
   </div>
 
   <div class="govuk-main-wrapper ncce-confidence">
     <div class="ncce-confidence__card">
-      <%= image_pack_tag 'media/images/home/card_courses.jpg', loading: 'lazy', class: 'ncce-confidence__icon', alt: '' %>
+      <%= link_to image_pack_tag('media/images/home/card_courses.jpg', loading: 'lazy', class: 'ncce-confidence__icon', alt: ''), courses_path %>
       <%= link_to t("pages.home.confidence.courses.title"), courses_path, class: "ncce-confidence__link" %>
       <p class="ncce-confidence__description">
         <%= t("pages.home.confidence.courses.description") %>
       </p>
     </div>
 
     <div class="ncce-confidence__card">
-      <%= image_pack_tag 'media/images/home/card_funding.jpg', loading: 'lazy', class: 'ncce-confidence__icon', alt: '' %>
+      <%= link_to image_pack_tag('media/images/home/card_funding.jpg', loading: 'lazy', class: 'ncce-confidence__icon', alt: ''), bursary_path %>
       <%= link_to t("pages.home.confidence.funding.title"), bursary_path, class: "ncce-confidence__link" %>
       <p class="ncce-confidence__description">
         <%= t("pages.home.confidence.funding.description") %>
       </p>
     </div>
 
     <div class="ncce-confidence__card">
-      <%= image_pack_tag 'media/images/home/card_teaching_resources.jpg', loading: 'lazy', class: 'ncce-confidence__icon', alt: '' %>
-      <%= link_to t("pages.home.confidence.teaching_resources.title"), home_teaching_resources_path, class: "ncce-confidence__link" %>
+      <%= link_to image_pack_tag('media/images/home/card_teaching_resources.jpg', loading: 'lazy', class: 'ncce-confidence__icon', alt: ''), curriculum_key_stages_path %>
+      <%= link_to t("pages.home.confidence.teaching_resources.title"), curriculum_key_stages_path, class: "ncce-confidence__link" %>
       <p class="ncce-confidence__description">
         <%= t("pages.home.confidence.teaching_resources.description") %>
       </p>
     </div>
 
     <div class="ncce-confidence__card ncce-confidence__card--orange">
-      <%= image_pack_tag 'media/images/home/card_i_belong.jpg', loading: 'lazy', class: 'ncce-confidence__icon', alt: '' %>
-      <%= link_to t("pages.home.confidence.i_belong.title"), i_belong_path, class: "ncce-confidence__link" %>
+      <%= link_to image_pack_tag('media/images/home/card_i_belong.jpg', loading: 'lazy', class: 'ncce-confidence__icon', alt: ''), "/i-belong" %>
```

Comment:
Can you swap the hard coded i-belong link to use the `about_i_belong_path` helper method. We should use the path helpers where possible rather than hard coded urls.